app/controllers/laptop_reservations_controller.rb

```diff
@@ -1,6 +1,6 @@
 class LaptopReservationsController < ApplicationController
   before_action :authenticate_user!
-  before_action :set_laptop_reservation, only: %i[show update destroy]
+  before_action :set_laptop_reservation, only: %i[show destroy]
```

Comment:
Could you please explain why this line is needed?

Reply:
@shasherazi I am talking about line 3, sorry for the poor explanation in my comment.

Reply:
Yeah this line eliminates the need to do 
`@laptop_reservation = LaptopReservation.find(params[:id])`

in show and destroy methods. It was included by @monicah79 so I just left it in.

Reply:
yeah thanks my bad, I couldn't see the private method, I was just looking at the changes, not the actual file.

Reply:
@shasherazi And also for the future don't forget to mention me, so I can get the notification on time.